src/main/kotlin/de/bund/digitalservice/useid/identification/IdentificationSessionController.kt

```diff
@@ -52,6 +49,10 @@ class IdentificationSessionsController(private val identificationSessionService:
         content = [Content()],
     )
     @SecurityRequirement(name = "apiKey")
+    @Tag(
+        name = "Identification Sessions",
+        description = "An identification session represent an ongoing identification flow of a user and stores the required information.",
+    )
```

Comment:
I feel like this should stay on class level. I would vote for a separate `TCTokenController` to separate concerns here.

Reply:
Plus, if we wanna be consistent and use the new grouping by actor (see tag name "eID-Client" below) for our API docs, this tag name would need to be "eService".

Reply:
I agree, that having this on class level feels better, but thinking about e.g. the time-based tokens, we have two endpoints and each one for a different actor. So we have to decide: Grouping by context internally and by actor in the docs OR grouping by actor everywhere OR grouping by actor in the docs and otherwise a mix of both 😅. I'd actually vote for the first option and think it's not too confusing to do it like that. What do you think?

Still, for this specific case, I think it's valid to have its own TCTokenController separately! 👍

Reply:
Agree on everything.